### PR TITLE
Feature/card

### PR DESCRIPTION
**Pull request by kalleericson**
Card feature, as run by the Statistics project.
Please check structure is in line and OK.

### CODE REVIEW

File: src/components/ui/card/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
+import * as React from 'react';
```
Comment (knegusen) [inline src/components/ui/card/Card.tsx]: Could this one be split up in more components? It's quite big right now.

**Comment by kalleericson** [comment]
screen shot 2018-08-22 at 10 44 29

- Split "Card" component into subcomponents "Header, HeaderButton, HeaderNotice, Content". 
- Updated test suite to match new structure.
- Added screenshot

**Comment by kalleericson** [comment]
Yeah, I'm aware and have changed structure of all tests written _after_ our discussion. However, I didn't change the tests already written. From now on, all is in order!

Great!

**Comment by knegusen** [comment]
Merge this?

**Pull request merged**